tgui/packages/tgui/interfaces/PreferencesMenu/antagonists/antagonists/thief.ts

```diff
@@ -0,0 +1,22 @@
+import { Antagonist, Category } from "../base";
+import { multiline } from "common/string";
+
+export const THIEF_MECHANICAL_DESCRIPTION
```

Comment:
Why exporting twice instead of importing from the other one? That's why I did this pattern anyway